.ci/azure-pipelines.yml

```diff
@@ -1,76 +1,74 @@
-name: $(Date:yyyyMMdd)$(Rev:.r)
-
-pr:
-  autoCancel: true
-
 trigger:
   batch: true
+  branches:
+    include:
+    - '*'
+  tags:
+    include:
+    - '*'
+
+pr:
+  branches:
+    include:
+    - '*'
 
 jobs:
-  - job: docs_build
-    displayName: "Main Build"
+  - job: Build
+    displayName: 'Build'
+
     pool:
       vmImage: windows-latest
+
     steps:
     - checkout: self
       clean: true
       submodules: true
       persistCredentials: true
 
     - task: CmdLine@2
-      displayName: "Install Markdownlint CLI"
+      displayName: 'Install Markdownlint CLI'
       inputs:
         script: 'npm install markdownlint-cli'
         workingDirectory: '$(Build.SourcesDirectory)'
 
     - task: CmdLine@2
-      displayName: "Run Markdownlint"
+      displayName: 'Run Markdownlint'
       inputs:
         script: 'npx markdownlint **/*.md --ignore node_modules --ignore src'
         workingDirectory: '$(Build.SourcesDirectory)'
 
     - task: CmdLine@2
-      displayName: "Update Submodules"
+      displayName: 'Update Submodules'
       inputs:
         script: 'git submodule update --recursive --remote'
         workingDirectory: '$(Build.SourcesDirectory)'
 
     - task: DotNetCoreCLI@2
-      displayName: "Build Server Submodule"
+      displayName: 'Build Server Submodule'
       inputs:
         command: build
         arguments: '--configuration Release'
         workingDirectory: '$(Build.SourcesDirectory)/src/jellyfin'
 
     - task: PowerShell@2
-      displayName: "DocFX Build"
+      displayName: 'DocFX Build'
       inputs:
         targetType: 'filePath'
         filePath: '.ci/build-in-ci.ps1'
         workingDirectory: '$(Build.SourcesDirectory)'
 
     - task: ArchiveFiles@2
-      condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'BuildCompletion'))
-      displayName: "Package Documentation"
+      condition: contains(variables['Build.SourceBranch'], 'master')
+      displayName: 'Package Documentation'
       inputs:
         rootFolderOrFile: '$(Build.SourcesDirectory)/_site'
         includeRootFolder: false
-        archiveType: 'tar'
-        tarCompression: 'gz'
-        archiveFile: '$(Build.ArtifactStagingDirectory)/docs-$(Build.BuildId).tar.gz'
-        replaceExistingArchive: true
+        archiveType: 'zip'
+        archiveFile: '$(Build.ArtifactStagingDirectory)/jellyfin-docs-$(Build.BuildId).zip'
 
-    - task: GitHubRelease@0
-      condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'BuildCompletion'))
-      displayName: "Create GitHub Release"
+    - task: PublishBuildArtifacts@1
+      displayName: 'Publish Docs'
```

Comment:
This needs the same condition as the previous task. Otherwise there is nothing to publish.